sdk/rust-examples/prime-numbers-generator/remove-five-seven-multiples/src/main.rs

```diff
@@ -0,0 +1,49 @@
+//! Prime Numbers Generator : Remove Multiples of 5 or 7
+//!
+//! ## Context
+//!
+//! Remove all number that are multiples of 5 or 7
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use std::fs;
+use anyhow;
+
+const FILENAME: &'static str = "/output/number-set.txt"; 
+
+fn main() -> anyhow::Result<()>{
```

Comment:
Can these `remove-five-seven-mulitples` and `remove-two-three-multiples` programs be unified into a single program?  Again, by taking a list of divisors on the command line as arguments, we can have a single program that performs both tasks, then:

```
generate-numbers --upper-bound 256 | filter-divisors --divisors 5 7 | filter-divisors --divisors 2 3
```

achieving the same effect as before...

Reply:
I get your point. I did that to expand the steps so I can get at least three chained programs. Thus, I got this ugly structure, I will refactor it to a more pleasant implementation.

Reply:
What I am thinking to do is : I will generate a configurable set of numbers from the first program, and the second program will apply Sieve of Eratosthenes algorithms to filter out non-prime numbers and write out the output.